### DISCUSSION TITLE
Assign peak2gene to cell types.

### DISCUSSION THREAD

**Discussion by tzhu-bio** [GreenleafLab/ArchR#1366] [Questions / Documentation] [answered]
Hello!
Is there an easy way to assign the peak2gene links to cell types? I want to get celltype-specific links.

**Comment by rcorces** [comment, marked as answer]
you could overlap them with cell type-specific peaks from getMarkerFeatures

**Reply by tzhu-bio** [reply]
Thank you very much.
